core/frontend/src/assets/colors/vuetify.ts

```diff
@@ -0,0 +1,7 @@
+export const PRIMARY = '#1976D2'
```

Comment:
what is the difference between this colors and the ones defines by default ?

Reply:
They are the same. I'm adding them here to allow for easier usage, as Vuetify (as far as I could check) doesn't export them (as 'primary', 'secondary', etc) anywhere.